src/web/content/static/css/letterpress.css

```diff
@@ -579,6 +578,10 @@ a.sms-large-font:hover {
     width: auto;
 }
 
+.sms-results-filter-container {
+    justify-content: space-between;
```

Comment:
Makes sure Sort by is aligned all the way to the right.
![image](https://github.com/searchmysite/searchmysite.net/assets/1785755/df3485d0-fcc6-42eb-9215-3fd64458e5d7)